pkg/auth/login.go

```diff
@@ -135,7 +135,7 @@ func copyAndFilter(w io.Writer, r io.Reader) ([]byte, error) {
 func filteredInformation(s []byte) []byte {
 
 	// List of strings to correctly filter
-	s = bytes.Replace(s, []byte("new-project"), []byte("project create"), -1)
+	s = bytes.Replace(s, []byte("oc new-project"), []byte("odo project create"), -1)
```

Comment:
`oc` changed the message a little bit. Without this change, the `odo login` outputs `oc project create` instead of `odo project create` 
```
▶ odo login -u developer -p developer
Connecting to the OpenShift cluster

Login successful.

You don't have any projects. You can try to create a new project, by running

    oc project create <project-name>
```